src/service/performance-impl.js

```diff
@@ -97,7 +97,7 @@ export class Performance {
         });
 
     // Tick window.onload event.
-    loadPromise(win).then(() => {
+    loadPromise(win.document).then(() => {
```

Comment:
Why? Is there even a document onload event? Could you please provide a reference?